templates/guess_the_painter.html

```diff
@@ -1,37 +1,37 @@
 {% extends "layout.html" %}
 {% block body %}
 <body>
-    {% include "login_register_segment.html"%}
-    <div class="page">
-        <h1>PaintingTrivia</h1>
-        {% for message in get_flashed_messages() %}
-            <div class="flash">{{ message }}</div>
-        {% endfor %}
-        <div>Statistics:</div>
-        <div>Right guesses: {{ right_guesses }}</div>
-        <div>Wrong guesses: {{ wrong_guesses }}</div>
-        <div>Who is this painting created by?</div>
+{% include "login_register_segment.html"%}
+<div class="page">
+    <h1>PaintingTrivia</h1>
+    {% for message in get_flashed_messages() %}
+    <div class="flash">{{ message }}</div>
+    {% endfor %}
+    <div>Statistics:</div>
+    <div>Right guesses: {{ quiz.score|sum() }}</div>
```

Comment:
just create a method in quiz that is gonna provide you information about the right and wrong guesses. no need to put logic into templates